src/client/client_context.h

```diff
@@ -72,6 +72,7 @@ class ClientContext {
 
   std::shared_ptr<ClientDestination> CreateNewLocalDestination(
       bool is_public = false,
+      // TODO(anonimal): we don't need DSA-SHA1 for default
```

Comment:
Why not just change it then?
We should probably make the whole thing configurable.